client/signup/steps/domains/index.jsx

```diff
@@ -1182,6 +1182,10 @@ export class RenderDomainsStep extends Component {
 			return translate( 'Enter some descriptive keywords to get started' );
 		}
 
+		if ( stepSectionName === 'use-your-domain' ) {
```

Comment:
It would make sense to follow the convention similar to the condition below, where conditions are flipped.

Also, as we touch this part, we could fix the last condition to use the `stepSectionName` const there.